spykes/django-admin-school/src/user/models.py

```diff
@@ -58,9 +71,31 @@ def __str__(self):
 
 
 class Employee(AppUser):
-    extra = models.CharField(
-        verbose_name=_('extra'),
-        max_length=40
+    #vacation = models.ForeignKey(

```

Comment:
Please remove this field, if it needs to be seen in admin panel then other operations must be done